src/simplicial-hott/08-covariant.rzk.md

```diff
@@ -1056,24 +1306,6 @@ types as follows.
     ( representable-dhom-from-cofibration-composition A a x y f u)
 ```
 
-## Covariant lifts, transport, and uniqueness
-
-In a covariant family C over a base type A , a term u : C x may be transported
-along an arrow f : hom A x y to give a term in C y.
-
-```rzk title="RS17, covariant transport from beginning of Section 8.2"
-#def covariant-transport
```

Comment:
Why are you removing this?

Reply:
If  I go to my repository this is in my file, other things above it were removed. Again, don't know what is happening.

Reply:
It might be an issue about which branch you are.

Reply:
This is also still in that file, starting at line 702

Reply:
I see you moved it. Sorry you are right.